perun-core/src/test/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntryIntegrationTest.java

```diff
@@ -885,7 +905,7 @@ public void updateResource() throws Exception {
 		facility = setUpFacility();
 		resource = setUpResource();
 
-		Resource resourceToUpdate = resourcesManager.createResource(sess, resource, vo, facility);
```

Comment:
Well, you are right, that it's not needed for simple update use-case. But you added restriction, that two resources can't have same name and you don't test it here or in another test. So you actually need two resources to check new behavior of update method.

Please remove assigning ``Resource resourceToUpdate = resource;`` since they are the same object after that. You can use ``resource`` in all code to prevent confusion (setName() on resourceToUpdate but setDescription() on resource). Thank you.